docs/data/joy/integrations/icon-libraries/IconFontSizes.tsx

```diff
@@ -1,19 +1,22 @@
 import * as React from 'react';
-import { useTheme } from '@mui/joy/styles';
 import Stack from '@mui/joy/Stack';
 import Person from '@mui/icons-material/Person';
 
 export default function IconFontSizes() {
-  const theme = useTheme();
   return (
     <Stack
       spacing={2}
       direction="row"
       sx={{ gridColumn: '1 / -1', alignItems: 'center', justifyContent: 'center' }}
     >
-      {Object.keys(theme.fontSize).map((size) => (
-        <Person key={size} fontSize={size} />
-      ))}
+      <Person fontSize="xs" />
```

Comment:
also cc @siriwatknp for suggesting a different solution if necessary

https://deploy-preview-41248--material-ui.netlify.app/joy-ui/integrations/icon-libraries/#size